invenio_rdm_migrator/streams/records/models.py

```diff
@@ -55,6 +55,17 @@ class RDMParentMetadata:
     _table_name: InitVar[str] = "rdm_parents_metadata"
 
 
+@dataclass
+class RDMCommunityParentMetadata:
```

Comment:
dummy question: we are not migrating the [community_metadata](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/communities/records/models.py#L23) table but only the parent? That would go to a `communities` stream at some point, no?

Reply:
yeah I might need to rename that to
```suggestion
class RDMParentCommunityMetadata:
```